### ISSUE TITLE
where is libfilesystem source?

### ISSUE THREAD

**Issue by lifning** [devkitPro/libnds#43]
i'm building the toolchain and libs myself on a non-x86 host, and some the nitrofs-related project in nds-examples can't find libfilesystem.

cheers

**Comment by fincs** [comment]
https://github.com/devkitPro/libfilesystem

**Comment by WinterMute** [comment]
Ideally you shouldn't be building the toolchain and libraries yourself. 

What non-x86 host are you using and what OS? Please be as specific as you can. We have plans to extend support for at least one non-x86 host in the near future.

**Comment by lifning** [comment]
my main dev machine has a POWER9 CPU in ppc64le mode, running Void Linux.

my laptop is a Pinebook Pro (aarch64), also running Void Linux.  it's more of a nice-to-have at this point, as far as using it to develop goes.

to be quite honest, for my desktop, i'm more comfortable compiling this stuff myself than i would be installing yet another package manager - it's got threads for days and doesn't take long at all, and i don't imagine many other folks are rocking a PowerPC for their homebrew workstation to make it worth your while to support.  the laptop, on the other hand, well, i imagine with the popularity of 64-bit ARM surging lately with single-board computers, Chromebooks, Linux4Tegra on Switch, etc. there'd be an audience worth supporting, on devices with few enough CPU cores that pre-builts would be more appreciated. (probably easier to come by ARM64 build machines too...)

**Comment by Oreo639** [comment]
You don't need to compile the libraries like libnds and libfilesystem, not to mention they are stored in a seperate dkp-libs repo. Also, everything is installed to `/opt/devkitpro` so it isn't going to conflict with the system package manager.

The only things that need to be recompiled are the actual binaries that run on the system.

I did make this a while ago, although that does remind me that I should PR pacman to the void-packages repo.
https://github.com/Oreo639/void-unsanitary-packages

If I made a mistake feel free to correct me.
